torch/nn/modules/conv.py

```diff
@@ -138,6 +138,8 @@ def __init__(self,
         self.reset_parameters()
 
     def reset_parameters(self) -> None:
+        # Setting a=sqrt(5) in kaiming_uniform is the same as initializing with
+        # uniform(-1/sqrt(k), 1/sqrt(k)), where k = weight.size(1) * prod(*kernel_size)
```

Comment:
Can you also add a line to the effect of: "See <link> for more details." where link points to Soumith's comment explaining the calculation (https://github.com/pytorch/pytorch/issues/15314#issuecomment-477448573)